tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+import pytest
+from fastapi.testclient import TestClient
+
+from api.app import app
+
+@pytest.fixture
+def test_client():
+    """Returns a Starlette test API instance"""
+    return TestClient(app)
```

Comment:
for test files, must they not start with test_?

Reply:
conftest.py is a special file in Pytest where you can set up fixtures/configs for tests.

https://www.tutorialspoint.com/pytest/pytest_conftest_py.htm

Reply:
I'm going to write some tests and I might use the existing fixture and/or add others.

Reply:
interesting might be good to add that link as a comment.
Thanks for sharing!

Reply:
Added a comment at the top :)